pages/pgql-1.1-spec.md

```diff
@@ -1489,6 +1489,21 @@ SELECT p1.name
 
 Above, we compare two string properties from different graphs. Besides properties, it is also possible to compare vertices and edges from different graphs. However, because PGQL 1.1 does not have concepts like graph views, base graphs, or sharing of vertices/edges between graphs, such comparisons will always yield `false`.
 
+## Subqueries Inside PATH Clause
+
+Users can add a sub-query in the `WHERE` clause of the `PATH` definition. One might be intereseted in asserting for specific properties for a node in the `PATH`. The following example defines a path ending in a node which is not the oldest in the graph:
```

Comment:
intereseted -> interested
node -> vertex (for consistency)   3x

Reply:
done.